### PR TITLE
Added 2018 PbPb collision vertex smearing parameters

### PR DESCRIPTION
**Pull request by ColeLindsey**
Added in vertex smearing parameters from the 2018 PbPb collision. More detailed presentation can be seen at:

https://cernbox.cern.ch/index.php/s/J3Emh3kw0Y3EGmf

### CODE REVIEW

**Comment by lpernie** [comment]
please test

File: Configuration/StandardSequences/python/VtxSmeared.py [hunk 1]
```diff
@@ -58,7 +58,7 @@
     'Realistic100ns13TeVCollisionBetaStar90mLowBunches' : 'IOMC.EventVertexGenerators.VtxSmearedRealistic100ns13TeVCollisionBetaStar90mLowBunches_cfi',
     'Realistic25ns13TeVEarly2017Collision' : 'IOMC.EventVertexGenerators.VtxSmearedRealistic25ns13TeVEarly2017Collision_cfi',
     'Realistic25ns13TeVEarly2018Collision' : 'IOMC.EventVertexGenerators.VtxSmearedRealistic25ns13TeVEarly2018Collision_cfi',
-
+    'RealisticPbPbCollision2018' : 'IOMC.EventVertexGenerators.VtxSmearedRealisticPbPbCollision2018_cfi.py',
```
Comment (lpernie) [inline Configuration/StandardSequences/python/VtxSmeared.py]: Should be _cfi, ad in the lines above

File: IOMC/EventVertexGenerators/python/VtxSmearedParameters_cfi.py [hunk 2]
```diff
@@ -713,6 +713,19 @@
     Z0 = cms.double(0.645)
 )
 
+# From fit to 5 TeV PbPb data
```
Comment (mmusich) [inline IOMC/EventVertexGenerators/python/VtxSmearedParameters_cfi.py]: @ColeLindsey it is considered good practice to document in this file how the numbers reported in the configuration have been obtained by breaking down the contribution from the Beam Spot and the pixel barycenter positions.
You might refer to this:
https://github.com/cms-sw/cmssw/blob/f084ec018ce6e9998d89201674f4a9fe37465fea/IOMC/EventVertexGenerators/python/VtxSmearedParameters_cfi.py#L597-L612
as a good example.

**Comment by tocheng** [comment]
@ColeLindsey 
Could you please update the PR based on @lpernie and @mmusich 's comments?
Thanks!

**Comment by civanch** [comment]
please test

**Comment by civanch** [comment]
+1

**Comment by fabiocos** [comment]
+operations

addition of a new beam spot scenario, no change to the previous ones

**Comment by fabiocos** [comment]
+1

**Pull request merged**